artman/config/proto/config_pb2.py

```diff
@@ -0,0 +1,569 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I know we did this for toolkit in Java with a gradle plugin. Do you happen to know whether protobuf python has such feature or not?